bin/score.py

```diff
@@ -19,9 +20,31 @@ def score_submission(predictions_path: str, status: str) -> dict:
         score_status = "INVALID"
         score1, score2, score3 = None, None, None
     else:
+        # Unzipping the predictions and extracting the files in
+        # the current working directory
+        if ".zip" in os.path.basename(predictions_path):
+            with zipfile.ZipFile(predictions_path, 'r') as zip_ref:
+                for zip_info in zip_ref.infolist():
+                    if zip_info.is_dir():
+                        continue
+                    # Extract the file ignoring directory structure it was zipped in
+                    zip_info.filename = os.path.basename(zip_info.filename)
+                    zip_ref.extract(zip_info, os.getcwd())
+
+        # Grabbing the extracted predictions files
+        predictions_files = glob.glob(os.path.join(os.getcwd(), "*.csv"))
+
+        # Checking if there are any files
+        if len(predictions_files) == 0:
+            score_status = "INVALID"
+            message = "No predictions files found"
+            score1, score2, score3 = None, None, None
```

Comment:
Maybe out of scope for this ticket - But i'd assume if there are not any prediction files you'd want to return right here and not execute any code below this.

Reply:
I updated this `score.py` so that it can handle zip files and be consistent with `validate.py`, but the `predictions_files` don't actually end up affecting the score, so this is really just placeholder code. That being said, I can update the `score.py` once I get more context from CNB on how the scores are calculate, but will leave that out of this PR to avoid scope creep.